admin/views/school/select-pupils-not-taking-check.ejs

```diff
@@ -35,37 +35,95 @@
+                            </ul>
+                        </li>
+                    </ul>
+                </details>
+            </div>
+
+            <br class="clearfix"/>
+            <br class="clearfix"/>
+
+            <h2>2. Select pupil(s)</h2>
+            <% if (pupilsList && pupilsList.length > 0) { %>
+            <table class="spacious sorting" id="pupilsList" role="listbox" aria-label="Select pupils.">
+                <thead>
+                <tr>
+                    <th scope="col">
+                        <a href="/school/pupils-not-taking-check/select-pupils/name/<%= htmlSortDirection.name %>">Pupil name <span class="<%= arrowSortDirection.name %>"><span>Sort by pupil name <%= htmlSortDirection.name %></span></span></a>
+                    </th>
+                    <th scope="col">
+                        <a href="/school/pupils-not-taking-check/select-pupils/reason/<%= htmlSortDirection.reason %>">Reason <span class="<%= arrowSortDirection.reason %>"><span>Sort by reason <%= htmlSortDirection.reason %></span></span></a>
+                    </th>
+                    <th scope="col">
+                        <div class="tick-all-checkboxes-wrapper">
+                            <label class="tick-all-checkboxes" id="selectAll" for="tickAllCheckboxes">Select all</label>
+                            <label class="tick-all-checkboxes all-hide" id="unselectAll" for="tickAllCheckboxes">Unselect all</label>
+                            <div class="multiple-choice-mtc">
+                                <input id="tickAllCheckboxes" name="allPupils" type="checkbox" aria-label="Select all pupils.">
+                                <div></div>
+                            </div>
+                        </div>
+                    </th>
+                </tr>
+                </thead>
+                <tbody>
+                    <% pupilsList.forEach( (pupil, index) => { %>
+                    <tr>
+                        <td>
+                            <label for="pupil-<%= pupil._id %>">
+                                <%= pupil.lastName %>, <%= pupil.foreName %>
+                            </label>
+                        </td>
+                        <td>
+                            <%= pupil.reason %>
+                        </td>
+                        <td>
+                            <div class="multiple-choice-mtc">
+                                <input id="pupil-<%= pupil._id %>"
+                                       name="pupil[<%= index %>]"
+                                       type="checkbox"
+                                       value="<%= pupil._id %>"
+                                       aria-label="Tick pupil <%= pupil.foreName %> <%= pupil.lastName %>."
+                                       aria-checked="false"
+                                       role="checkbox" />
+                                <div></div>
```

Comment:
Should not need this empty `div`

Reply:
Please see previous comment. I'm open to other solutions if you have a better one!